### PR TITLE
Release ripple-binary-codec version 1.3.2

### PR DESCRIPTION
**Pull request by khancode**
## High Level Overview of Change

Update package.json for release

### Type of Change

- [x] Release

### CODE REVIEW

**Review by mvadari — APPROVED** [review]

**Review by intelliot — APPROVED** [review]

**Review by intelliot — CHANGES_REQUESTED** [review]
Sorry, just noticed I already took the 1.3.1 version number. Make this 1.3.2

File: package-lock.json [hunk 1]
```diff
@@ -16652,7 +16652,7 @@
       }
     },
     "packages/ripple-binary-codec": {
-      "version": "1.3.0",
+      "version": "1.3.1",
```
Comment (intelliot) [inline package-lock.json:16655]:
```suggestion
      "version": "1.3.2",
```

File: packages/ripple-binary-codec/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ripple-binary-codec",
-  "version": "1.3.0",
+  "version": "1.3.1",
```
Comment (intelliot) [inline packages/ripple-binary-codec/package.json:3]:
```suggestion
  "version": "1.3.2",
```

**Review by intelliot — CHANGES_REQUESTED** [review]

**Comment by mvadari** [comment]
I think @khancode just forgot to push his changes - this was done when we published last night

**Review by intelliot — APPROVED** [review]

**Pull request merged**